x-pack/plugins/fleet/common/openapi/bundled.yaml

```diff
@@ -2623,6 +2631,20 @@ components:
           type: string
         version:
           type: string
+        release:
+          type: string
+          enum:
+            - experimental
+            - beta
+            - ga
+        source:
+          type: object
+          properties:
+            license:
+              type: string
+              enum:
+                - Apache-2.0
+                - Elastic-2.0
```

Comment:
are these the only possible values?

Reply:
yes per the current spec: https://github.com/elastic/package-spec/blob/73b30e11123f7db583b376ff0a9516f9362dcd28/spec/integration/manifest.spec.yml#L145